features/support/paths.rb

```diff
@@ -609,8 +609,11 @@ def path_to(page_name, *args)
     when /^the integration show page for (service ".+?")/
       admin_service_integration_path(Transform $1)
     when /^the integration page for (service ".+?")/
+      # TODO: THREESCALE-3759 edit page no longer exist, remove and replace for show
       edit_admin_service_integration_path(Transform $1)
     when 'the service integration page'
+      # TODO: THREESCALE-3759 edit page no longer exist, change for
+      # settings_admin_service_path(provider_first_service!)
```

Comment:
```suggestion
      # TODO: THREESCALE-3759 edit page no longer exist, change
```

Reply:
Y U NO like this comment? Is it wrong?

Reply:
the thing is that this `the service integration page` is not used anymore from anywhere, so why replace it for X when we can just remove it because it is simply not used at all? 😄

Reply:
ahh true, then it should be removed altogether.